packages/storybook/stories/va-card.stories.tsx

```diff
@@ -58,3 +65,9 @@ withIcon.argTypes = {
     description: 'Displays an icon at the top of the card in a blue circle. Value is the icon name.',
   },
 };
+
+export const withCallToAction = Template.bind(null);
+withCallToAction.args = {
+  ...defaultArgs,
+  'has-cta': true,
+};
```

Comment:
> [!note]
> Added this story to card because it seemed like an unwrapped va-link inside a card was not abnormal in the VA.gov ecosystem. Wanted to have a story so that we can track visual regressions and make a11y testing easier.